server/seed.js

```diff
@@ -0,0 +1,60 @@
+var faker = require('faker');
+const mongoose = require('mongoose');
+const { Question } = require('../database/index.js');
+const { User } = require('../database/index.js');
+const { Answer } = require('../database/index.js');
+
+// var randomUserName = faker.name.findName();
+// var randomDate = faker.date.past();
+// var randomSentence = faker.lorem.sentence();
+// var randomProfilePic = faker.image.imageUrl();
+// //PERHAPS COME BACK TO CHANGE IMAGES TO IMAGES OF PEOPLE
+// //WASN'T SURE IF MONGO DB COULD STORE IMAGES OR JUST URLS
+// var randomLocation = faker.address.city() + ', ' + faker.address.country();
+// var numContributions = Math.floor(Math.random() * 200);
+// var numHelpfulVotes = Math.floor(Math.random() * 50);
+// var numLikes = Math.floor(Math.random() * 20);
+
+//For loop, Function that generates data, invoke the function, Another function inserts that data into the database
+
+var createNewData = () => {
+  var newQuestion = new Question ({
+    date: faker.date.past(),
+    question: faker.lorem.sentence()
+  });
+  newQuestion.save((err) => {
+    if (err) {
+      console.log('THIS WAS AN ERROR')
```

Comment:
you might want to console.log the particular err here
also, one thing you could do is write the save functionality as its own function inside of your database index.js, and then you wouldn't have to write out the save bit for Question, User, and Answer (similar to how we did it in the first mongoose sprint) this doesn't save too much code, though, so tradeoff may not be worth it

Reply:
Good points! I will incorporate